api/apps/api/src/modules/geo-feature-tags/dto/update-geo-feature-tag.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsUUID, ValidateNested } from 'class-validator';
+import { TagInfoDTO } from '@marxan-api/modules/geo-feature-tags/dto/tag-info.dto';
+import { Type } from 'class-transformer';
+
+export class UpdateGeoFeatureTagDTO {
+  @ApiProperty()
+  @IsUUID()
+  projectId!: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @ValidateNested({ each: true })
+  @Type(() => TagInfoDTO)
+  tagInfo!: TagInfoDTO;
```

Comment:
This I was wasn't 100% convinced on, but it come from one comment in our early conversations about the high level design, where we could reuse the DTO, so that it wouldn't be needed to copy/paste all the validations in any new DTOs that require tag information. For example the message for the length validation would get duplicated but I guess we can turn it into a constant/function?